eva_cttv_pipeline/clinvar_xml_utils.py

```diff
@@ -346,25 +346,40 @@ def nsv_id(self):
             return None
 
     @property
-    def hgvs(self):
+    def _hgvs_elems(self):
         return [
-            elem.text
+            elem
             for elem in find_elements(self.measure_xml, './AttributeSet/Attribute')
             if elem.attrib['Type'].startswith('HGVS')
         ]
 
+    @property
+    def hgvs(self):
+        return [elem.text for elem in self._hgvs_elems]
+
+
+    @property
+    def refseq_hgvs(self):
+        refseq_elems = [elem for elem in self._hgvs_elems if 'LRG' not in elem.attrib['Type']]
+        # TODO determine if there is a preferred ordering to use here, based e.g. on prevalence in data
+        return [elem.text for elem in refseq_elems]
```

Comment:
Based on your comment https://github.com/EBIvariation/eva-opentargets/issues/262#issuecomment-885477481, wouldn't it be better to always catch the identifier whose ``Type`` is ``HGVS, genomic, top level``, so that we are consistent? As it is, wouldn't we be using different identifiers based on a (possibly) random order of the identifiers within the ``AttributeSet``?

Reply:
Yes you're right, I was thinking if they were interchangeable that I could get away with just picking the first one arbitrarily, but probably best to be consistent.

The only reason I even bothered to filter for Refseq is because of the field name we're using [here](https://github.com/EBIvariation/eva-opentargets/blob/master/consequence_prediction/repeat_expansion_variants/pipeline.py#L99), otherwise I think even LRG would be okay.